schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+          },
+          {
+            "name": "replaced_by",
+            "datatype": "string"
+          },
+          {
+            "name": "replaced_list",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          },
+          {
+            "name": "species",
+            "datatype": "string"
+          },
+          {
+            "name": "interpreted_record",
+            "datatype": "string"
+          },
+          {
+            "name": "included_record",
+            "datatype": "string"
+          },
+          {
+            "name": "variation_id",
+            "datatype": "integer"
+          },
+          {
+            "name": "variation_name",
+            "datatype": "string"
+          },
+          {
+            "name": "variation_type",
+            "datatype": "string"
+          },
+          {
+            "name": "date_created",
+            "datatype": "date"
+          },
+          {
+            "name": "date_last_updated",
+            "datatype": "date"
+          },
+          {
+            "name": "accession",
+            "datatype": "string"
+          },
+          {
+            "name": "version",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_of_submitters",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_of_submissions",
+            "datatype": "integer"
+          },
+          {
+            "name": "record_type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_record_history",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          },
+          {
+            "name": "accession",
+            "datatype": "string"
+          },
+          {
+            "name": "version",
+            "datatype": "integer"
+          },
+          {
+            "name": "date_changed",
+            "datatype": "date"
+          },
+          {
+            "name": "variation_id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "type_comment",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "data_source",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_comment_type",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "species",
```

Comment:
:+1: to having this in here for now, but it feels like something that'd be nice to move into an ontology store in the longer-term